spec/services/cfe/service_set_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+# rubocop:disable RSpec/VerifiedDoubles
```

Comment:
Rather than disabling this rule (what's the point of it if we disable it), could we define a class in the test?

Something like this might work?

```ruby
class TestObject
  def passported?; end

  def non_passported?; end
end

let(:object) { instance_double(TestObject, passported?: true, non_passported?: false) }
```

Although to be honest I think some of the RSpec rubocop rules are a arbitrary/a waste of time/annoying anyway, so I wouldn't mind if you just turned it off completely 🤣

Reply:
It's only disabled for this spec.
I went through several iterations with rubocop complaining of various things at various points. This cop seemed the least applicable in the context of this spec. I did look at using an anonymous class but though the effort and complextity is not worth it (your example would raise another cop violation i believe). We do not need to verify a fake class IMO overall, so disabled it